box_types_metadata.go

```diff
@@ -167,6 +175,85 @@ func (sd *StringData) StringifyField(name string, indent string, depth int, ctx
 	return "", false
 }
 
+// Item is a item under an item list
+// https://developer.apple.com/documentation/quicktime-file-format/metadata_item_list_atom/item_list
+type Item struct {
+	AnyTypeBox
+	Version  uint8   `mp4:"0,size=8"`
+	Flags    [3]byte `mp4:"1,size=8"`
+	ItemName []byte  `mp4:"2,size=8,len=4"`
+	Data     Data    `mp4:"3"`
+}
+
+// StringifyField returns field value as string
+func (i *Item) StringifyField(name string, indent string, depth int, ctx Context) (string, bool) {
+	switch name {
+	case "ItemName":
+		return fmt.Sprintf("\"%s\"", util.EscapeUnprintables(string(i.ItemName))), true
+	}
+	return "", false
+}
+
 func isUnderIlstFreeFormat(ctx Context) bool {
 	return ctx.UnderIlstFreeMeta
 }
+
+func BoxTypeKeys() BoxType { return StrToBoxType("keys") }
+
+func init() {
+	AddBoxDef(&Keys{})
+}
+
+/*************************** keys ****************************/
```

Comment:
💭 I feel good about the addition of the `Keys` BlockType below but am unsure on the `Item` above AnyBlockType changes (see discussion https://github.com/abema/go-mp4/pull/159/files#r1438420071). I'm happy to split the PR into 2 to merge in the `Keys` changes if more work is iteration is needed on the number `Item` changes.